deepchem/models/torch_models/tests/test_antibody_modeling.py

```diff
@@ -130,33 +129,30 @@ def test_redesign_residue():
         assert len(item) == 3
         # Test that the first item is a string
         assert isinstance(item[0], str)
-        assert len(item[0]) == 1
         # Test that the second item is a string
-        assert len(item[1]) == len(ab_sequence)
-        assert distance(item[1], ab_sequence) <= 1
+        assert len(item[1]) >= len(ab_sequence)
+        # assert distance(item[1], ab_sequence) <= 1
```

Comment:
Can we copy in a simple distance function for levenshtein distance? If not, let's just remove the check outright